src/triage.rs

```diff
@@ -52,11 +52,11 @@ pub async fn pulls(
     }
 
     let mut pulls: Vec<Value> = Vec::new();
-    for base_pull in base_pulls.into_iter() {
+    for base_pull in base_pulls.into_iter().filter(|pull| !pull.draft) {
```

Comment:
Instead of excluding draft PRs, it might be a better idea to just mark them in gray. WG-triage doesn't triage draft waiting-on-review and waiting-on-author PRs, though it seems we do triage draft blocked PRs. T-clippy might triage draft PRs as well.

Reply:
Hm, I think it would make sense to triage them after some time to close them out (basically draft is similar to waiting-on-author I think, in fact I might say that we should just treat it equivalently)...

Reply:
> Hm, I think it would make sense to triage them after some time to close them out

Yeah, they probably should continue to be included.

> basically draft is similar to waiting-on-author I think, in fact I might say that we should just treat it equivalently

I wouldn't say they're equivalent, in fact I would say most of the time they are different. For many PRs, waiting-on-author means "the author needs to address reviewer comments or fix conflicts". On the other hand, draft means "this is only partially implemented".

Reply:
+1 for hiding or indicating as a draft, I'd like to have an option to hide draft (and even `S-blocked`) since we don't have to check them frequently and rust-lang/rust's triage queue is quite long. I think something like this would be a good way here (this just hides PRs but we can tweak attributes, e.g., change the color): https://github.com/JohnTitor/triagebot/tree/hide-drafts